### PR TITLE
update

### PR DESCRIPTION
**Pull request by JonasTrampe**
Update/Deploy to a fixed path "/opt/hucon"
Copy existing programs to "/root/hucon/code"

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -12,26 +12,44 @@ opkg update
...
-# add the server to start at boot
-sed -i '/start_server.sh/d' /etc/rc.local
-sed -i "/^exit 0/i sh /opt/hucon/start_server.sh >> /tmp/hucon.log 2>&1 & " /etc/rc.local
+echo "Deploying new server startup"
+cp /opt/hucon/init.d/HuconWebserver /etc/init.d/HuconWebserver
+chmod +x /etc/init.d/HuconWebserver
+/etc/init.d/HuconWebserver enable
+
+echo "Removing old i2c startup"
+if [[ -e /etc/init.d/I2C_led ]] || [[ -e /etc/init.d/I2C_led ]]; then
```
Comment (SaschaMzH) [inline install.sh]: Did you mean the following:
`if [[ -e /etc/init.d/i2c_led ]] || [[ -e /etc/init.d/I2C_led ]]; then`

**Both checks are uppercase**

File: update.sh [hunk 2]
```diff
@@ -56,10 +56,16 @@ if [ $do_update ]; then
...
 
         # download the new package
-        downloadUrl="https://github.com/juwis/hackerschool/releases/download/$latestVersion/hucon.run"
+        downloadUrl="https://github.com/juwis/hackerschool/releases/download/$latestVersion/hucon-$latestVersion.run"
 
         # Download the new package
-        wget $downloadUrl
+        wget $downloadUrl -o hucon.run
+
+        echo "Check if existing code can be moved to /root/hucon/code..."
```
Comment (SaschaMzH) [inline update.sh]: What about the **examples** folder inside the code folder which will be copied?

**Review by SaschaMzH — CHANGES_REQUESTED** [review]

File: install.sh (hunk 1 above)
Comment (JonasTrampe) [reply install.sh]: yes you are right - pasta in the copier is not good xD

File: update.sh (hunk 2 above)
Comment (JonasTrampe) [reply update.sh]: old examples will be deleted and the software only sources the new ones

**Review by SaschaMzH — APPROVED** [review]

**Pull request merged**